utils/build-script-impl

```diff
@@ -1235,9 +1237,7 @@ function calculate_targets_for_host() {
             android-*)
                 swift_sdk="ANDROID"
                 build_for_this_target=$(not ${SKIP_BUILD_ANDROID})
-                # FIXME: Allow Android host tests to be enabled/disabled by the
-                #        build script.
-                test_this_target=
+                test_this_target=$(not ${SKIP_TEST_ANDROID_HOST})
```

Comment:
Since #2880 just landed, I believe there will need to be an update to the matching logic in `build-script`.